### ISSUE TITLE
NordVPN bug blocking transfers

### ISSUE THREAD

**Issue by cwaynedavis1946** [BiglySoftware/BiglyBT#2377]
NordVPN update broke torrenting with BiglyBT. All transfers failed, got this error message:
ERROR: socket Selector.open() for 'TCP network manager reader' failed 10 times in a row, aborting.
This App / Java is likely being firewalled!

Reported to Nord, sent diagnostics. They recommended connection changes that didn't help. Finally trying to pin it down, disabled split tunnelling and solved the problem. Informed Nord so they hopefully will fix the bug. Whined about them not properly testing updates.

**Comment by cwaynedavis1946** [comment]
NordVPN asked me multiple times to generate more diagnostics, claiming they could not duplicate the problem.  I got fed up and sent this to them:
------------------------------------------------------------------
Please review previous emails on this issue.

I took a spare PC, reformatted the drive then did a clean Windows 10 install.  Updated Windows.  Downloaded and installed BiglyBT and NordVPN.  Started BiglyBT with default configuration, started download of Ubuntu iso.  Transfer worked.

Configured NordVPN for split tunneling with VPN only for BiglyBT and Edge.  Connected to P2P server with VPN.  Started BiglyBT.  Transfers would not start.  NordVPN bug behaving exactly as I initially reported.  This is the 3rd Windows 10 PC I have tried and the bug happened on all 3.

If your support staff cannot duplicate this bug then they are totally incompetent.  Instead I'm certain that they haven't even tried, just as implied in their first message.

I purchased a 2 year subscription for SurfShark and configured it for split tunneling.  Testing showed that it is performing properly and BiglyBT transfers work.  Today and tomorrow I will be replacing NordVPN with SurfShark on my other machines.  If I should encounter any problems with SurfShark I'm sure that their support staff will actually provide some support.

**Comment by lislei** [comment]
I'm curiuos to ask - did you hear back from them? :)

**Comment by cwaynedavis1946** [comment]
I heard from Nord several times, still claiming that they couldn't
duplicate the bug.  Finally I reinstalled NordVPN, reproduced the bug and
sent them registry settings and VPN diagnostics they had asked for.
Nothing from them since, they have not fixed the bug.

SurfShark caused problems with RDC and local file transfers so I switched
to PIA.  It has an option for local LAN connections to bypass the VPN  so
RDC and local file connections don't go through the VPN.  Working like a
charm for everything.  Wish I'd gone to it from the start instead of using
NordVPN until the bug happened
